src/openxr/OpenXRApi.cpp

```diff
@@ -1534,21 +1505,26 @@ void OpenXRApi::uninitialize() {
 	if (session != XR_NULL_HANDLE) {
 		xrDestroySession(session);
 		session = XR_NULL_HANDLE;
+
+		for (XRExtensionWrapper * wrapper : registered_extension_wrappers) {
+			wrapper->on_session_destroyed();
+		}
```

Comment:
I wonder if we need to call the extension code before actually destroying the session, we may be cleaning things up that still need to have the session available. Not sure to be honest...

Reply:
Since the opposite callback is invoked after the session is initialized, it makes sense to have this one invoked prior to the session being destroyed. I'll swap the order.